Sources/SwiftTileserverCache/WebServer.swift

```diff
@@ -90,24 +90,26 @@ public class WebServer {
             let width = UInt16(request.parameters["width"] ?? ""),
             let height = UInt16(request.parameters["height"] ?? ""),
             let scale = UInt8(request.parameters["scale"] ?? ""),
+            let orientation = UInt8(request.parameters["orientation"] ?? ""),
+            let tilt = UInt8(request.parameters["tilt"] ?? ""),
```

Comment:
move `orientation` and `tilt` from here to just bellow the guard and give them an default value of 0 ` ?? 0`